openhands_cli/tui/settings/store.py

```diff
@@ -33,6 +34,38 @@ def load_mcp_configuration(self) -> dict[str, Any]:
         except Exception:
             return {}
 
+    def load_skills(self) -> list:
+        """Load skills from user directories and project-specific directories."""
+        all_skills = []
+
+        # Load user skills from ~/.openhands/skills and ~/.openhands/microagents
+        try:
+            user_skills = load_user_skills()
+            all_skills.extend(user_skills)
+        except Exception:
+            pass
+
+        # Load project-specific skills from .openhands/skills and legacy microagents
+        project_skills_dirs = [
+            Path(WORK_DIR) / ".openhands" / "skills",
+            Path(WORK_DIR) / ".openhands" / "microagents",  # Legacy support
```

Comment:
nit: later down the road, we can consider supportinga  nested git repo for this

eg a git repo under workdir which has its own `skills` / `microagents`